src/bzmesh.jl

```diff
@@ -139,14 +140,12 @@ function bzmesh_ir_wedge(kgrid_size, lattice, atoms; tol_symmetry=1e-5)
     end
 
     if !isempty(kreds_notmapped)
-        eirreds, esymops = find_irreducible_kpoints(kreds_notmapped, Stildes, τtildes)
-
-        @info("$(length(kreds_notmapped)) reducible kpoints could not be generated from " *
-              "the irreducible kpoints returned by spglib. $(length(eirreds)) of " *
-              "these are added as extra irreducible kpoints.")
-
-        append!(kirreds, eirreds)
-        append!(ksymops, esymops)
+        error("$(length(kreds_notmapped)) reducible kpoints could not be generated from " *
+              "the irreducible kpoints returned by spglib.")
+        # # add them as reducible anyway
+        # eirreds, esymops = find_irreducible_kpoints(kreds_notmapped, Stildes, τtildes)
+        # append!(kirreds, eirreds)
+        # append!(ksymops, esymops)
```

Comment:
I would vote to still keep this, even if it should not happen. If you are in a production calculation and the code crashes because of this you are not happy, believe me.

Reply:
Well we should fix it first :p but if something that should not happen happens, I have zero confidence in the result and would rather like it crashes than produce potentially bad results. We are not particularly production oriented anyway...